components/event-bus/cmd/event-bus-publish-knative/handlers/publish.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 var (
-	defaultChannelNamespace = "kyma-system"
+	DefaultChannelNamespace = "kyma-system"
```

Comment:
As the **DefaultChannelNamespace** became public, I suggest to move it somewhere under **internal/knative**.

Reply:
Done